netbox_netisp/forms.py

```diff
@@ -28,6 +28,7 @@ class Meta:
             "street_number",
             "street_name",
             "street_suffix",
+            "street_ordinance",
```

Comment:
Logically, the street_direction should come before street name. By adjusting the position in the array, you will adjust the position it is displayed in the form.